exercises/joseph_ramsay/ex02_oo_basics/src/circle.h

```diff
@@ -0,0 +1,20 @@
+#ifndef CIRCLE_H
+#define CIRCLE_H
+
+#include "shape.h"
+
+class Circle : public Shape {
+
+private:
+  int m_radius;
+
+public:
+  Circle(int radius);
+  int getNumSides() const;
```

Comment:
make use of `override`. See [here](http://en.cppreference.com/w/cpp/language/override). This is true for the other classes too ( but I won't mention it there)